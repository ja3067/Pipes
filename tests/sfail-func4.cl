def foo(x):
	return x + y

def bar(y):
	foo(y)

bar(3)
