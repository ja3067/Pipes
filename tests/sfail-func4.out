SNameError: name 'y' is not defined
